sdcm/utils/raft/__init__.py

```diff
@@ -18,6 +18,12 @@ class Group0MembersNotConsistentWithTokenRingMembersException(Exception):
 class RaftFeatureOperations(Protocol):
     _node: 'BaseNode'
 
+    expected_timeout_begin_message = 1200
+    expected_timeout_end_message = 7200
```

Comment:
where those times are coming from ?

Reply:
In the referenced bug report the decommission operation took about 40 minutes.
So, `1h` timeout for the `end` messages look ok.
But `20 minutes` for the `begin` one is too much. It must be much smaller. At least `5m`...